database/seed.js

```diff
@@ -17,38 +15,39 @@ var reviewSchema = new Schema({
   housingId: Number,
   reviewId: Number,
   name: String,
-  date: String,
-  displayDate: String,
+  date: Date,
   comment: String,
-  imgURL:String,
+  imgURL: String,
   ratings: {
     communication: Number,
     accuracy: Number,
     location: Number,
     checkIn: Number,
     cleanliness: Number,
     value: Number
-    }
-  });
-
+  }
+});
 
 var Review = mongoose.model('Reviews', reviewSchema);
 
 var rating = () => {
   var ratings = [1, 2, 3, 4, 5];
 
   return ratings[Math.floor(Math.random() * ratings.length)];
-}
+};
+
+var date = () => {
+  return faker.date.between('2008-09-01', '2019-10-30');
+};
 
 var save = () => {
-  for (var i = 0; i < 10; i++) {
+  for (var i = 0; i < 100; i++) {
     const filter = {reviewId: i};
     const update = {
       housingId: 12345,
       reviewId: i,
```

Comment:
I might have misunderstood this, but I thought we were supposed to have 100 primary records (i.e. housingIds), and then some variation of secondary records e.g. 5-20 reviewIds per housingId. This way e.g. if we go to out combined detail page for housingId 10, every module will have related data for that house. Might be worth double checking this with Maia/sheps

Reply:
To my understanding, we are basing each module off of the same housingId (1 listing to be compared to) with 100 sample reviews, but I may have misunderstood. I agree, let's ask Maia/sheps as this is an easy fix.

Reply:
Yea, the reason I mention is because when I talked to her about it for my module she said 100 housingIds and 3-5 photos for each (she said I could rotate sets of photos)